controllers/BrowseController.php

```diff
@@ -56,4 +60,37 @@ public function renderFileList($filesOrder = null, $foldersOrder = null)
         ]);
     }
 
+    public function actionLoadEntry()
+    {
+        $fileId = Yii::$app->request->get('id');
+        $fileId = strpos($fileId, 'file_') === 0 ? substr($fileId, 5) : 0;
+
+        if ($fileId && ($file = File::findOne(['id' => $fileId]))) {
```

Comment:
Can you please add some `canView()` (Content record) or `->readable()` (ActiveQueryUser) to check access oermssions?

Reply:
I added the `->readable()` in commit https://github.com/humhub/cfiles/pull/131/commits/eda8f380dcd41d9f30cd753ddf260220fd13733f